samples: nrf9160: Increase MQTT keepalive timeout (#3400)
nrfconnect/sdk-nrf: samples/nrf9160/cloud_client/README.rst

.. note::
   To output data in the terminal window located in the `nRF Cloud`_ web interface, the data format must be in JSON format.

> .. warning::
> 
>   The sample sets the option :option:`CONFIG_MQTT_KEEPALIVE` to the maximum allowed value specified by the configured cloud backend.
>   This is to limit the IP traffic between the device and the cloud providers message broker in order to encourage a low power sample.
>   However, it should be noted that in certain LTE networks, the NAT timeout can be conciderable lower than the maximum allowed MQTT keepalive. So as a recommendation, and to prevent the likelyhood of
>   getting thrown out of the connection "unexpectedly", option :option:`CONFIG_MQTT_KEEPALIVE` should be set to the lowest value of the aforementioned timeout limits. (Maximum allowed MQTT keepalive and NAT timeout).
> 

Reviewer: ```suggestion
.. note::
   The sample sets the option :option:`CONFIG_MQTT_KEEPALIVE` to the maximum allowed value that is specified by the configured cloud backend.
  This is to limit the IP traffic between the device and the message broker of the cloud provider for supporting a low power sample.
   However, note that in certain LTE networks, the NAT timeout can be considerably lower than the maximum allowed MQTT keepalive.
   So as a recommendation, and to prevent the likelihood of getting disconnected unexpectedly, the option :option:`CONFIG_MQTT_KEEPALIVE` must be set to the lowest of the aforementioned timeout limits (Maximum allowed MQTT keepalive and NAT timeout).
```
